src/test/java/guitests/ContextMenuTests.java

```diff
@@ -97,4 +113,42 @@ public void test3() {
         sleep(EVENT_DELAY);
     }
 
+    /**
+     * Marks and tests issues in a panel of 12 issues
+     * from Issue #{index} in the UI right to the end of the list as read/unread
+     * @param isMarkAsRead If true, tests whether issues on/below the selected issue are being correctly marked read
+     *                      If false, tests whether issues on/below the selected are being correctly marked unread
+     * @param index The issue number in the panel
+     */
+    private void markAndVerifyIssuesBelow(int index, boolean isMarkAsRead){
+        click("#dummy/dummy_col0_" + index);
+        rightClick("#dummy/dummy_col0_" + index);
```

Comment:
I don't like how `"#dummy/dummy_col0_"` is being repeated a lot, all over the place. Is there a way to avoid it?